modules/post/windows/capture/keylog_recorder.rb

```diff
@@ -5,7 +5,7 @@
 
 class MetasploitModule < Msf::Post
   include Msf::Post::Windows::Priv
-  include Msf::Post::File
+  include Msf::Post::Windows::Process
```

Comment:
The `Msf::Post::File` include is still required here because of the use of `file_local_write`.

```suggestion
  include Msf::Post::Windows::Process
  include Msf::Post::File
```